client/src/styles/LeftAndMain_Subsites.css

```diff
@@ -0,0 +1,82 @@
+/**
+* Styling for the subsite actions section in the CMS
+*/
+#SubsiteActions {
+  position: absolute;
+  padding: 0;
+  margin: 0;
+  right: 0;
+  top: 0;
+  width: 350px;
+  text-align: right;
+  margin-right: 130px;
+  height: 51px;
+  border-bottom: 3px solid #d4d0c8;
+  color: #fff;
+}
+
+#SubsiteActions fieldset {
+  padding: 3px;
+  border-style: none;
+  margin-top: 1px;
+  background: none;
+}
+
+#SubsiteActions fieldset span {
+  padding: 3px;
+}
+
+.cms-menu .cms-subsites {
+  padding: 3px 0 15px;
+}
+
+.cms-menu .cms-subsites .field.dropdown {
+  margin: 0 10px;
+  padding-bottom: 0;
+}
+
+.cms-menu.collapsed .cms-subsites {
+  display: none;
+}
+
+.cms-menu .cms-subsites .dropdown span {
+  padding-left: 5px;
+  color: black;
+}
+
+.cms-subsites .chosen-container-single .chosen-single div b {
+  margin: 0 5px;
+}
+
+#AddSubsiteLink {
+  display: block;
+  font-size: 80%;
+  margin-left: 3px;
+}
+
+#Form_AddSubsiteForm .field {
+  margin-left: 100px;
+}
+
+#Form_AddSubsiteForm label.left {
+  float: left;
+  width: 100px;
+  margin-left: -100px;
+}
+
+body.SubsiteAdmin .right form #URL .fieldgroup * {
+  font-size: 11px;
+}
+
+.cms-add-form #PageType li .class-SubsitesVirtualPage {
+  background-position: 0 -32px !important;
```

Comment:
would it be possible to do this without using !important?

Reply:
Also not a part of this PR. File is simply moved from original location.
https://github.com/silverstripe/silverstripe-subsites/blob/2/css/LeftAndMain_Subsites.css#L72